SPECS/envoy/envoy.spec

```diff
@@ -0,0 +1,440 @@
+#
+# spec file for package envoy-proxy
+#
+# Copyright (c) 2021 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%define _dwz_low_mem_die_limit  20000000
+%define _dwz_max_die_limit     100000000
+
+%define src_install_dir /usr/src/%{name}
+
+Name:           envoy
+Version:        1.14.4
+Release:        3.4%{?dist}
+Summary:        L7 proxy and communication bus
+License:        Apache-2.0
```

Comment:
Please use [Fedora's short license names](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files). In this case it might be "ASL 2.0".